app/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,7 @@
 
     <string name="appName" translatable="false">DuckDuckGo</string>
     <string name="duckDuckGoLogoDescription">DuckDuckGo logo</string>
-    <string name="appDescription">Privacy, simplified</string>
+    <string name="appDescription" translatable="false">DuckDuckGo&#160;</string>
```

Comment:
We add a blank space at the end so the description shows in Android 10. If the app name is the same as the description then no description is shown. Bing used to show both but now it doesn't (I take due to this).